taipy/core/taipy.py

```diff
@@ -238,19 +241,23 @@ def submit(
             in asynchronous mode.
         timeout (Union[float, int]): The optional maximum number of seconds to wait
             for the jobs to be finished before returning.
+        **properties (dict[str, any]): A keyworded variable length list of additional arguments.
```

Comment:
This information is not really useful. What happens with these 'additional arguments'?

Reply:
hm you're right, I don't have much with me atm. Can you suggest something? Would something like this work?
```suggestion
        **properties (dict[str, any]): A keyworded variable length list of additional arguments that will be stored within the `Submission^`.
```

Reply:
Yeah why not, if that's what's going on. May be a link to the API to retrieve that from the Submission object would be even better...

Reply:
you can retrieve it from the submission object :D

Reply:
Sure. That's my point.
Replacing the \`Submission^\` xref by \`Submission.get_properties()^\` or whatever.